Hapi 13 peerDependencies Support?

Hi, we were hoping to upgrade from Hapi 11 to 13 and receive the following error running `npm install`:

```
npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/bflad/.nvm/versions/node/v4.4.1/bin/node" "/Users/bflad/.nvm/versions/node/v4.4.1/bin/npm" "install"
npm ERR! node v4.4.1
npm ERR! npm  v2.14.20
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package hapi@13.4.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer reptile@2.1.1 wants hapi@>=11.x.x
```

The peerDependencies are currently too strict to support both 11 and 13:

```
"peerDependencies": {
  "hapi": ">=11.x.x"
},
```

I noticed you already have 13.x.x in the devDependencies, are you planning on cutting a release with 13 support? Please let me know if you need any help testing or upgrading. Thanks!

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,4 @@
+*
+!lib/
+!package.json
+!LICENSE

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "hoek": "4.x.x"
   },
   "peerDependencies": {
-    "hapi": ">=11.x.x"
+    "hapi": "13.x.x"
   },
   "devDependencies": {
     "code": "3.x.x",

